core/src/components/select/select.tsx

```diff
@@ -82,7 +82,10 @@ export class Select implements ComponentInterface {
   @Prop({ reflect: true }) color?: Color;
 
   /**
-   * A property name or function used to compare object values
+   * This property allows developers to specify a custom function or property
+   * name for comparing objects when determining the selected option in the
+   * ion-select. When not specified, the default behavior will use object
```

Comment:
Same as other comment: I think this is called "strict equality" according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality